frontend_tests/zjsunit/namespace.js

```diff
@@ -171,7 +204,7 @@ exports._finish_template_mocking = () => {
 };
 
 exports._mock_template = (fn, exercise_template, f) => {
-    const path = "../../static/templates/" + fn;
+    const path = "../../" + TEMPLATE_PATH + fn;
```

Comment:
I think this has `..//static/templates`.

Reply:
Fixed.